region/tourian/main/Metroid Room 2.json

```diff
@@ -931,6 +984,63 @@
         "Shoot the door open and spark out."
       ]
     },
+    {
+      "link": [2, 1],
+      "name": "Come in Shinecharged, Leave With Spark (Left Side, Tricky Dash Jump)",
+      "entranceCondition": {
+        "comeInShinecharged": {}
+      },
+      "requires": [
+        {"shineChargeFrames": 130},
+        "canTrickyDashJump",
+        "canShinechargeMovementTricky",
+        {"or": [
+          {"and": [
+            {"or": [
+              "f_KilledMetroidRoom2",
+              "ScrewAttack",
+              {"and": [
+                {"metroidFrames": 145},
+                {"or": [
+                  "Plasma",
+                  "Wave",
+                  {"ammo": {"type": "Missile", "count": 1}}
```

Comment:
By the looks of it, these don't work with a super locked door.  Or would be adding extra requirements.

Reply:
Actually, extra requirements would be correct for strat repeatability.

Reply:
Yeah the `unlocksDoors` here already requires `f_KilledMetroidRoom2` or `ScrewAttack`, which makes the Plasma/Wave/Missile option irrelevant.